src/scripts/eos-analysis_TEST.d/analysis.yaml

```diff
@@ -79,3 +79,28 @@ predictions:
       - name: B->pilnu::P(q2_min,q2_max)
         kinematics: { 'q2_min': 0.01, 'q2_max': 2.00 }
       - name: B_u->lnu::BR
+
+steps:
+  - name: optimize
+    iterations:
+      - posterior: CKM+FF
+        seed: 123
+    tasks:
+      - task: find-mode
+        arguments:
+          label: 'seed-{seed}'
+
+  - name: sample
+    iterations:
+      - { posterior: CKM+FF }
+    tasks:
+      - task: sample-mcmc
+        arguments:
+          chain: '0,1-2,3'
+          pre_N: 500
```

Comment:
Can we input the seed here is as well? I want to check that the analysis is indeed reproducible.

Reply:
The task automatically assigns a seed based on the chain index:

https://github.com/eos/eos/blob/67aa0916a26270d0fe8bb3e08b534160e7bef4bf/python/eos/tasks.py#L217

Reply:
hmm true, should we maybe move all the seed assignment to the arguments of the task?

Reply:
For ``sample-mcmc`` I would be opposed. Here, seeding is achieved through the chain index. Let's not make it a 2D problem.